enhancements/network/allow-from-router-networkpolicy.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Allow-From-Router" NetworkPolicy
+authors:
+  - "@danwinship"
+reviewers:
+  - "@squeed"
+  - "@trozet"
+  - "@Miciah"
+approvers:
+  - "@knobunc"
+  - "@squeed"
+creation-date: 2020-12-10
+last-updated: 2020-12-10
+status: implementable
+---
+
+# "Allow-From-Router" NetworkPolicy
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Operational readiness criteria is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Many users want to create NetworkPolicies that say, effectively,
```

Comment:
Does this proposal make sense for upstream kube? I would imagine there is value in having something like this in upstream network policies for host-network traffic in general.

Reply:
the upstream kube version of supporting host-network traffic would be the `nodeSelector` proposal linked below